tests/e2e/setup_for_test.go

```diff
@@ -131,37 +130,36 @@ func configQedServer(id int, pathDB, signPath, tlsPath string, tls bool) *server
 // 	- the second one deletes the server the first one created
 // Each server instance is completely new and blank.
 // It will also generate all the needed keys for the instance.
-func newServerSetup(id int, tls bool) (func() error, func() error) {
+func newServerSetup(id int, tls bool) (func() (string, error), func() error) {
```

Comment:
I prefer newServerSetup to return (path, before, after) instead of returning path on each before() call and ignore it all the time.